content/source/docs/enterprise-beta/sentinel/enforce.html.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: enterprise2
+page_title: "Sentinel - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-sentinel-enforce"
+---
+
+# Enforce and Override Policies
+
+Once a policy is added to an organization it is enforced on all subsequent Workspace Runs.
+
+The policy check will occur immediately after a plan is successfully executed in the run. If the plan fails, the policy check will not be performed. The policy check uses the tfplan generated in the plan of the run along with the state and configuration to verify the rules in each of the policies.
```

Comment:
The policy check uses the generated tfplan file along with the state and configuration to verify the rules in each of the policies.